svf-llvm/include/SVF-LLVM/LLVMModule.h

```diff
@@ -87,6 +87,9 @@ class LLVMModuleSet
     LLVMType2SVFTypeMap LLVMType2SVFType;
     Type2TypeInfoMap Type2TypeInfo;
 
+    FunDeclToDefMapTy AppFuncDef2SVFExternFuncDef;
+    FunDeclToDefMapTy SVFExternFuncDef2AppFuncDef;
```

Comment:
Remove this map. Take a look at createSVFDataStructure in LLVMModule.cpp. If you have two def functions with the same name "foo"  in app and "svf_foo" in ext.c. We will map both llvm functions to the same svf function named "svf_foo" if we would like use "svf_foo" to overwrite "foo"

Reply:
make sure both llvm function maps to the same svf function in this map: LLVMFunc2SVFFunc